scripts/src/javatests/org/oppia/android/scripts/label/AccessibilityLabelCheckTest.kt

```diff
@@ -0,0 +1,353 @@
+package org.oppia.android.scripts.label
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.oppia.android.testing.assertThrows
+import java.io.ByteArrayOutputStream
+import java.io.PrintStream
+
+/** Tests for [AccessibilityLabelCheck]. */
+class AccessibilityLabelCheckTest {
+  private val outContent: ByteArrayOutputStream = ByteArrayOutputStream()
+  private val originalOut: PrintStream = System.out
+  private val ACCESSIBILITY_LABEL_CHECK_PASSED_OUTPUT_INDICATOR = "ACCESSIBILITY LABEL CHECK PASSED"
+  private val ACCESSIBILITY_LABEL_CHECK_FAILED_OUTPUT_INDICATOR = "ACCESSIBILITY LABEL CHECK FAILED"
+
+  @Rule
+  @JvmField
+  public var tempFolder = TemporaryFolder()
+
+  @Before
+  fun setUp() {
+    tempFolder.newFolder("testfiles")
+    System.setOut(PrintStream(outContent))
+  }
+
+  @After
+  fun restoreStreams() {
+    System.setOut(originalOut)
+  }
+
+  @Test
+  fun testAccessibilityLabel_labelPresent_activitiesAreDefinedWithAccessibilityLabel() {
+    val manifestContent =
+      """
+      <?xml version="1.0" encoding="utf-8"?>
+      <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        package="org.oppia.android">
+
+        <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+        <uses-permission android:name="android.permission.INTERNET" />
+        <application
+          android:name=".app.application.OppiaApplication"
+          android:allowBackup="true"
+          android:icon="@mipmap/ic_launcher"
+          android:label="@string/app_name"
+          android:roundIcon="@mipmap/ic_launcher_round"
+          android:supportsRtl="true"
+          android:theme="@style/OppiaTheme">
+          <meta-data
+            android:name="firebase_analytics_collection_deactivated"
+            android:value="true" />
+          <meta-data
+            android:name="firebase_crashlytics_collection_enabled"
+            android:value="false" />
+          <meta-data
+            android:name="automatic_app_expiration_enabled"
+            android:value="false" />
+          <meta-data
+            android:name="expiration_date"
+            android:value="2020-09-01" />
```

Comment:
I suggest removing all the cruft that's not actually needed for the purpose of these tests.

Reply:
Removed.